setup.py

```diff
@@ -1546,6 +1546,19 @@ def build_extensions(self):
               'pyarrow>=0.15.0',  # Petastorm 0.7.7 is not compatible with < 0.15.0
               'pyspark>=2.3.2'
           ],
+          'tensorflow':  [
+              'tensorflow'
+          ],
+          'tensorflow-gpu':  [
+              'tensorflow-gpu'
+          ],
+          'pytorch':  [
+              'torch',
```

Comment:
You mean `horovod[torch]`?  I was debating whether to call it `horovod[torch]` or `horovod[pytorch]`.  I went with pytorch because `HOROVOD_WITH_PYTORCH=1` is the env variable we decided to use, but we can change it if you feel it would make more sense as torch.

Reply:
I had in my branch:
```
          'pytorch': [
              'pytorch',
              'torchvision'
          ],
```
What is the right dependency, `torch` or `pytorch`?

Reply:
I don't know what the difference is though: https://stackoverflow.com/questions/44371560/what-is-the-relationship-between-pytorch-and-torch
your decission

Reply:
I get an error when attempting to `pip install pytorch`:

```
Exception: You tried to install "pytorch". The package named for PyTorch is "torch"
```

This seems to be the preferred name.

Reply:
said thing that it says pytorch everywhere in the docs and code, maybe we should stick to pytorch and rename / refactor this in a separate PR, maybe never...

Reply:
Sounds good, we'll use pytorch.